### PR TITLE
Fix HTTP data source not passing along query string parameters

### PR DESCRIPTION
**Pull request by pianomansam**

### CODE REVIEW

**Comment by pianomansam** [comment]
The above commits will cause the tests to not pass, demonstrating the issue. Once demonstrated, I'll commit a fix.

File: packages/appsync-emulator-serverless/httpSource.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 const fetch = require('node-fetch');
 
 const httpSource = async (endpoint, { resourcePath, method, params }) => {
-  const response = await fetch(endpoint + resourcePath, {
+  const { query } = params || {};
+  const queryPath =
+    query === undefined ? '' : `?${new URLSearchParams(Object.entries(query))}`;
```
Comment (cbaron) [inline packages/appsync-emulator-serverless/httpSource.js]: curious why you wrapped `query` with Object.entries.  [Here](https://nodejs.org/api/url.html#url_class_urlsearchparams) it says URLSearchParams accepts an object.

Comment (pianomansam) [reply packages/appsync-emulator-serverless/httpSource.js]: Good call. Just committed an update to remove that.

**Comment by pianomansam** [comment]
> One question, also thinking we need a version bump in package.json.

I attempted to bump the version, but Travis CI didn't like it.
https://travis-ci.org/ConduitVC/aws-utils/jobs/459859998

**Comment by cbaron** [comment]
@pianomansam -- my guess is that's because we stick to `MAJOR.MINOR.REVISION`.

I think this calls for a REVISION increment by 1.

**Pull request merged**